src/executor_cleanup.c

```diff
@@ -30,92 +30,26 @@ t_err	ft_cleanup_cmd(t_cmd *cmd)
 {
 	t_err	err;
 
-	err = ft_close(cmd->fd_in);
+	err = ft_close(&cmd->fd_in);
 	if (err != SUCCESS)
 		return (err);
-	err = ft_close(cmd->fd_out);
+	err = ft_close(&cmd->fd_out);
 	if (err != SUCCESS)
 		return (err);
 	err = ft_plug_pipe(cmd);
 	if (err != SUCCESS)
 		return (err);
 	if (cmd->heredoc)
 		ft_unlink_heredoc(&cmd->heredoc, err);
+	if (cmd->delims)
+		ft_free_str_arr(cmd->delims);
+	if (cmd->args)
+		ft_free_str_arr(cmd->args);
+	if (cmd->outfiles)
+		ft_free_str_arr(cmd->outfiles);
```

Comment:
Outfile bool array also needs to be free'd

Reply:
supplemented